modules/tensor_mechanics/src/materials/PowerLawCreepStressUpdate.C

```diff
@@ -32,24 +32,24 @@ validParams<PowerLawCreepStressUpdate>()
   params.addParam<Real>("start_time", 0.0, "Start time (if not zero)");
   params.addCoupledVar("temperature", 0.0, "Coupled temperature");
   params.addParam<std::string>(
-      "creep_prepend", "", "String that is prepended to the creep_strain Material Property");
+      "creep_prepend", "creep", "String that is prepended to the creep_strain Material Property");
 
   return params;
 }
 
 PowerLawCreepStressUpdate::PowerLawCreepStressUpdate(const InputParameters & parameters)
-  : RadialReturnStressUpdate(parameters, "creep"),
+  : RadialReturnStressUpdate(parameters, parameters.get<std::string>("creep_prepend")),
```

Comment:
`getParam()` is a member function on `MooseObject()`, it's a convenience member you use so that you don't have to write `parameters.get()`all the time. The reason it doesn't work here is because the object is being constructed (e.g. `MooseObject` doesn't really exist yet). 

This is one of the reasons we don't really like to change constructors from the MOOSE standard, but it is what it is here.